[CI]: examples fmt-check fails (3 files unformatted) blocks release-plz Release PR #4671

## Affected Workflow

`examples-test.yml` (called via `examples-test-standalone.yml`)

## Issue Description

`examples-test-standalone` jobs `examples-tutorial-basis` and `examples-twitter` fail at the **Check Format** step (`cargo make fmt-check`, which runs `cargo fmt --all -- --check`).

This currently blocks the release-plz Release PR #4671 (`chore: release` for `0.1.0-rc.30`) from passing CI.

## Steps to Reproduce

1. From the repository root, run the format check inside each example workspace:
   ```bash
   cd examples/examples-tutorial-basis && cargo fmt --all -- --check
   cd examples/examples-twitter        && cargo fmt --all -- --check
   ```
2. Both report `Diff in ...` for files listed below and exit non-zero.

## Error Logs

```text
Examples Tests / examples-tutorial-basis  Error: Execution error: Some files are not properly formatted
Examples Tests / examples-tutorial-basis  Summary: 1 would be formatted, 49 unchanged, 0 ignored, 0 errors
Examples Tests / examples-tutorial-basis  Error while executing command, exit code: 1
Examples Tests / examples-tutorial-basis  ##[error]Process completed with exit code 105.

Examples Tests / examples-twitter  Error: Execution error: Some files are not properly formatted
Examples Tests / examples-twitter  Summary: 1 would be formatted, 103 unchanged, 0 ignored, 0 errors
Examples Tests / examples-twitter  Error while executing command, exit code: 1
Examples Tests / examples-twitter  ##[error]Process completed with exit code 105.
```

Specifically:

- `examples/examples-tutorial-basis/src/apps/polls/client/components.rs` — `use` declaration order needs to be re-sorted (typed-URL `links` alias must follow the `server_fn::{...}` group).
- `examples/examples-twitter/src/apps/relationship/client/components.rs` — trailing IIFE arg list should collapse to a single line.
- `examples/examples-twitter/src/core/client/components/common.rs` — `page!(|| { div { class: \"divider\" } })()` should collapse to a single line.

None of these files are touched by the Release PR #4671 diff (it only edits version metadata + CHANGELOGs + `examples/CLAUDE.md`), so the drift was introduced by earlier merges into `main`.

## Runner OS

- ubuntu-latest

## Workflow Run URL

- examples-tutorial-basis: https://github.com/kent8192/reinhardt-web/actions/runs/26226341479/job/77181466640
- examples-twitter:        https://github.com/kent8192/reinhardt-web/actions/runs/26226341479/job/77181466651

## Local Test Results

Reproduced locally on macOS (Darwin 25.4.0) with Rust 1.91.1 by running `cargo fmt --all -- --check` inside each `examples/examples-*` workspace.

Fix: run `reinhardt-admin fmt-all` (or `cargo fmt --all`) inside both `examples/examples-tutorial-basis` and `examples/examples-twitter`.

## Impact Assessment

- [x] Affects specific workflows only
- [x] Blocks all pull requests from merging (release-plz Release PR #4671 is currently red on this check)
- [ ] Intermittent failure (not consistent)
- [ ] Workaround available

## Additional Context

- Discovered while diagnosing failing checks on release-plz Release PR #4671.
- Likely introduced by one of the recent merges into `main` touching the `examples/` tree.

--- a/examples/examples-tutorial-basis/src/apps/polls/client/components.rs
+++ b/examples/examples-tutorial-basis/src/apps/polls/client/components.rs
@@ -36,11 +36,11 @@ use reinhardt::pages::resolve_static;
 // Typed URL helpers are now emitted by `#[url_patterns]` directly
 // (issue #4656); we alias the macro-emitted `urls` module as `links` to
 // keep call sites concise.
-use crate::apps::polls::urls::client_router::urls as links;
 use crate::apps::polls::server_fn::{
 	create_choice, create_question, delete_choice, delete_question, get_question_detail,
 	get_question_results, get_questions, submit_vote, update_choice, update_question,
 };
+use crate::apps::polls::urls::client_router::urls as links;
 
 /// Polls index page - List all polls
 ///

--- a/examples/examples-twitter/src/apps/relationship/client/components.rs
+++ b/examples/examples-twitter/src/apps/relationship/client/components.rs
@@ -134,11 +134,7 @@ pub fn follow_button(target_user_id: Uuid, is_following_initial: bool) -> Page {
 					}
 				}
 			}
-		})(
-			is_following_signal,
-			toggle_follow,
-			toggle_follow_for_error,
-		)
+		})(is_following_signal, toggle_follow, toggle_follow_for_error)
 	}
 
 	#[cfg(native)]

